src/main.ts

```diff
@@ -32,7 +33,7 @@ export async function createProject(options: Options) {
     {
       title: `🚀 Creating a new Scaffold-ETH 2 app in ${chalk.green.bold(
         options.project
-      )}`,
+      )}${options.template ? ` with the ${chalk.green.bold(getArgumentFromTemplateOption(options.template))} template ` : ""}`,
```

Comment:
template -> extension? Or not? Redundant space at the end btw

Reply:
Removed the space.

And yep, will be renaming `template` in the code.